docs/cep/query-guide/functions/geospatial/crosses.md

```diff
@@ -24,25 +24,25 @@ Determines if the specified object or location crosses a geographic location spe
 ## Example 1
 
 ```sql
-CREATE STREAM StreamA (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double);
-CREATE SINK STREAM OutputStream (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double, prediction double, confidence double);
-
-@info(name = 'basicBayesianRegressionQuery')
-FROM StreamA#streamingml:bayesianRegression('model1', attribute_0, attribute_1, attribute_2, attribute_3)
-INSERT ALL EVENTS INTO OutputStream;
+SELECT geo:crosses('km-4354', -0.5, 0.5, '{"type":"Polygon","coordinates":[[[0, 0],[2, 0],[2, 1],[0, 1],[0, 0]]]}') AS crosses;
 ```
 
-The `basicBayesianRegressionQuery` uses a Bayesian linear regression model named `model1` to predict the label of the feature vector represented by `attribute_0`, `attribute_1`, `attribute_2`, and `attribute_3`. The predicted value and the prediction confidence (inverse of the standard deviation of the predictive distribution) are emitted to the `OutputStream` stream, along with the feature vector. As a result, the `OutputStream` stream is defined as follows: `(attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double, prediction double, confidence double)`.
+This query uses the `geo:crosses()` function to determine if a line, defined by the coordinates `(-0.5, 0.5)` and `(km-4354)`, crosses the specified polygon. The polygon is represented as a GeoJSON string with coordinates `[[0, 0], [2, 0], [2, 1], [0, 1], [0, 0]]`.
 
 ## Example 2
 
 ```sql
-CREATE STREAM StreamB (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double, attribute_4 double);
-CREATE STREAM DetailedOutputStream (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double, attribute_4 double, prediction double, confidence double);
+CREATE STREAM InputStream (lineStart double, lineEnd double);
+CREATE STREAM OutputStream (crosses bool);
```

Comment:
@diana-macrometa - The same issue as yesterday with the `SINK` required for the `OutputStream`.